docker/dockerfile/Dockerfile

```
@@ -32,6 +25,19 @@ RUN apt-get update && apt-get install -y \
 ENV PATH="/usr/lib/postgresql/18/bin:${PATH}"
 
 
+FROM runtime-base AS build-base
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)